rktlet/image/imagestore.go

```diff
@@ -77,49 +80,71 @@ func (s *ImageStore) ImageStatus(ctx context.Context, req *runtime.ImageStatusRe
 		return nil, err
 	}
 
+	reqImg := req.GetImage().GetImage()
```

Comment:
The `.Get` things actually handle nil pretty gracefully/consistently. In the generated code:

https://github.com/kubernetes/kubernetes/blob/4adb23c8ccedd1819fc9396c75a28cf7f552bb59/pkg/kubelet/api/v1alpha1/runtime/api.pb.go#L1761-L1766

https://github.com/kubernetes/kubernetes/blob/4adb23c8ccedd1819fc9396c75a28cf7f552bb59/pkg/kubelet/api/v1alpha1/runtime/api.pb.go#L1090-L1095

So for all generated protobuf things, calling `((*ProtobufType)nil).GetFoo()` actually returns a nil `Foo` if the `ProtobufType` is nil itself... so it chains gracefully with no panics and you just have to check at the end.

Kinda like a non-obvious always-on nil coalescence operator.